crates/csslsrs/src/features/hover.rs

```diff
@@ -88,14 +90,13 @@ fn get_css_hover_content(
                     .flat_map(|props| props.iter())
                     .find(|prop| prop.name == name)
                 {
-                    return Some(format_css_entry(
+                    return Some(format_css_property_entry(
                         &property.name,
```

Comment:
would could pass the property in one go here instead of things one by one, should be fine (and might possibly be more performant?)